package.json

```diff
@@ -38,8 +38,8 @@
     "jest-environment-jsdom": "^29.7.0",
     "jsdom": "~24.0.0",
     "lint-staged": "^15.2.2",
-    "nx": "18.3.4",
-    "nx-cloud": "^18.0.1",
+    "nx": "18.1.1",
+    "nx-cloud": "18.0.0",
```

Comment:
Trying to fix an error with nx not working on my machine. I'll try again.

Reply:
I think the problem is with using pnpm v9 with nx. Unfortunately I upgraded pnpm to v9 (which significantly modifies the lock file) before realizing it's not yet supported by nx v18. Occasionally nx falls over and I have to restart my PC. I haven't yet figured out what triggers it. Perhaps we should do the work to go back to pnpm v8? Also nx v19.5 now supports pnpm v9 so that is likely the better path.